src/simulation/hdf_simulation_store.jl

```diff
@@ -330,6 +347,13 @@ end
 
 log_cache_hit_percentages(x::HdfSimulationStore) = log_cache_hit_percentages(x.cache)
 
+function _make_dataframe(data::Matrix{Float64}, columns::Tuple{Vector{String}})
+    if (ndims(data) < 2 || size(data)[1] == 1) && size(data)[2] != size(columns)[1]
```

Comment:
How could `ndims(data)` be anything other than 2?

Reply:
Sometimes it will be a a column vector